hw/ip/pinmux/data/pinmux_fpv_testplan.hjson

```diff
@@ -5,121 +5,795 @@
+            '''
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: DftJtagO1Backward_A
+      desc: '''`dft_jtag_o` pins are ones if one of the following conditions are met:
+            - Dft Jtag is enabled and the corresponding pins are 1.
+            '''
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+
+    {
+      name: TapStrap_A
+      desc: '''If `dft_hold_tap_sel_i` is 0 and `lc_dft_en_i` is On (tn the past two clock cycles
+            due to the synchronizer), or `strap_en_i` is 1.
+            And in the meantime, if `lc_hw_debug_en_i` is On (at the past two clock cycles due to
+            the synchronizer), then tap_strap must be equal to the past value of corresponding
+            `mio_in_i`.
+            '''
+      milestone: V1
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: TapStrap0_A
+      desc: '''If `dft_hold_tap_sel_i` is 0 and `lc_dft_en_i` is On (at the past two clock cycles
+            due to the synchronizer), or `strap_en_i` is 1.
+            Then tap_strap[0] must be equal to the past value of corresponding `mio_in_i`.
+            '''
+      milestone: V1
+      tests: ["pinmux_assert"]
+    }
+
+    // Jtag pinmux output tests
+    {
+      name: LcJtagI_A
+      desc: '''When Lc tap is selected, the corresponding `mio_out_o` and `mio_out_oe` should be
+            equal to `lc_jtag_i`.'''
+      milestone: V1
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: RvJtagI_A
+      desc: '''When Rv tap is selected and `lc_hw_debug_en_i` is On (at the past two clock cycles
+            due to the synchronizer), the corresponding `mio_out_o` and `mio_out_oe` should be equal
+            to `rv_jtag_i`.'''
+      milestone: V1
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: DftJtagI_A
+      desc: '''When Dft tap is selected and `lc_dft_en_i` is On (at the past two clock cycles
+            due to the synchronizer), the corresponding `mio_out_o` and `mio_out_oe` should be equal
+            to `dft_jtag_i`.'''
+      milestone: V1
+      tests: ["pinmux_assert"]
+    }
+
+    // Dft `strap_test_o` tests
+    {
+      name: DftStrapTestO_A
+      desc: '''When `lc_dft_en_i` is On (at the past two clock cycles due to the synchronizer),
+            `dft_strap_test_o.valid` must be 1, and `dft_strap_test_o.straps` should be equal to
+            the corresponding `mio_in_i` index.'''
+      milestone: V1
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: DftStrapTestOValidStable_A
+      desc: "`dft_strap_test_o.valid` once set to 1 will stay high until reset."
+      milestone: V1
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: DftStrapTestOStrapStable_A
+      desc: "`dft_strap_test_o.valid` once set, `dft_strap_test_o.straps` should stay stable."
+      milestone: V1
+      tests: ["pinmux_assert"]
+    }
+
+    // USB related IOs.
+    // Current plan is to only check connectivity because usbdev design is fully verified
```

Comment:
Are these connectivity checks in pinmux_assert, or in the connectivity CSVs?

Reply:
It is just an assertion to check the connectivity, not the CSVs. Let me update the description to clear the ambiguity. Thanks for pointing that out.